src/bindings/html/view.js

```diff
@@ -16,6 +17,7 @@ const observerConfig = {
 export class View {
   constructor(doc) {
     this.doc = doc;
```

Comment:
Yeah, I haven't figured out how to deal with private members in this code yet.  Should we also have `_service`, `_observe` and `_disconnect`?  Or maybe put most of the method definitions in the constructor and use the closure to hide private members?